### ISSUE TITLE
Remove HTML from strings to ease translations?

### ISSUE THREAD

**Issue by lionkmp** [MX-Linux/mx-tweak#128]
Hi,
Would it be possible to remove the many HTML codes from the strings, to ease translation work, and make sure translators don't break that? Strings like this in the  "defaultlook.ui" file, however seems to be Qt specific file, so I'm not sure that can be removed:

<string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Customize the panel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>

(After next release, of course, to not break existing translations. ;-) It's just not convenient to work with:

![tweak](https://github.com/MX-Linux/mx-tweak/assets/5088657/a74618b1-a6e8-42e7-bf2d-0d698865091f)

Thank you in advance. Feel free to close this if you think this won't be changed.

**Comment by AdrianTM** [comment]
Yeah, it's possible, we were thinking about this, but as you mentioned it would invalidate the entire translations.
For new packages we are aware of this and try to avoid this kind of situation, but these packages are old and mostly translated.
